src/test/java/shop/VirtualItemTest.java

```diff
@@ -0,0 +1,19 @@
+package shop;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
```

Comment:
Remove testng annotations and use junit only

Reply:
done